packages/pathway-mapper/src/ui/Toolbar.tsx

```diff
@@ -108,9 +110,13 @@ export default class Toolbar extends React.Component<IToolbarProps, {}>{
           }}/>,
       
 
-          <img height="22px" width="22px" data-border="true" data-type="light" data-tip="Edit pathway" data-place="right" data-effect="solid" src={openImage} onClick={() => {{window.open("http://pathwaymapper.org/?pathwayName=" + this.props.selectedPathway +"&"+ studyQuery )}}}/>
+          !this.props.groupComparisonView && <img height="22px" width="22px" data-border="true" data-type="light" data-tip="Edit pathway" data-place="right" data-effect="solid" src={openImage} onClick={() => {{window.open("http://pathwaymapper.org/?pathwayName=" + this.props.selectedPathway +"&"+ studyQuery )}}}/>
           ])}
           <img height="22px" width="22px" data-border="true" data-type="light" data-tip="Help" data-place="right" data-effect="solid" src={aboutImage} onClick={() => {this.props.handleOpen(EModalType.CHELP); }}/>
+          { this.props.genesSelectionComponent &&
+            this.props.genesSelectionComponent(
+                )
+            }
```

Comment:
This is probably okay as a single line statement 
```suggestion
          {this.props.genesSelectionComponent && this.props.genesSelectionComponent()}
```